src/components/sections/TitleBar.jsx

```diff
@@ -53,6 +58,10 @@ export default class TitleBar extends Component {
       this.setState({
         docked: true,
       });
+
+      if (onHeightChanged) {
+        onHeightChanged(node.offsetHeight);
```

Comment:
Hmm, we're not actually notifying that the height has changed here. We're notifying that the title bar has docked, and passing the height at the time of docking. Which I think is fine -- but let's rename this prop to onDocked.

Reply:
Hey Ray, thanks for reviewing. In general, I will avoid hardcoding anything in the future. I look forward to reviewing your changes and let me know if there's anything I can do to help. Thanks again!